src/libmongoc/src/mongoc/mongoc-cmd.c

```diff
@@ -811,12 +834,14 @@ mongoc_cmd_parts_assemble (mongoc_cmd_parts_t *parts,
                            bson_error_t *error)
 {
    mongoc_server_description_type_t server_type;
+   mongoc_server_api_t *api;
```

Comment:
This is totally unrelated, but is there a reason to not declare `*api` scoped in the condition below? Is this a matter of preference or is this not supported in some C standards?

Reply:
It's a matter of preference for me, and is the style followed in this file. I like for my code additions to be consistent with existing styles in the surrounding code whenever it is reasonable to do so.  As far as I know C99 would allow the variable declaration at the beginning of any scope, though.

Reply:
I see, thanks for explaining!